src/components/Configure/actions/write/onSaveWriteCreateInstallation.ts

```diff
@@ -66,6 +66,12 @@ const generateCreateWriteConfigFromConfigureState = (
     },
   };
 
+  // insert proxy into config if it is enabled
+  const isProxyEnabled = hydratedRevision.content.proxy?.enabled;
```

Comment:
The added lines look fine (once `hydratedRevision.content.proxy?.enabled;` is put into a helper function), but I'm pretty confused by this `generateCreateWriteConfigFromConfigureState` function. 
- the file name is `onSaveWriteCreateInstallation` but the comment above the function refers to update installation request, is this an outdated comment?
- the comment line 59 says "need empty read.standardObjects for update read", what does this mean? Can you expand the comment line to be more clear?
- do you always add `read` and `write` keys to config.content even when the hydratedRevision doesn't have read or write actions? Have you found that things break if you don't add them?

Reply:
Yes, read/write was breaking on the backend if the other wasn't supplied. This may have been fixed @RajatPawar - did we have a ticket or did you fix this by chance?

Reply:
It looks like it's working now for create installation but then throws a null pointer exception during the read update. I've reported a bug for this if you did want to remove this hack. 

I've updated the comment in the code with the linear ticket for context. 
https://linear.app/ampersand/issue/ENG-780/bug-write-createupdate-installation-without-read